DMCompiler/DM/Visitors/DMVisitorProcBuilder.cs

```diff
@@ -59,6 +59,14 @@ class DMVisitorProcBuilder : DMASTVisitor {
             _proc.Continue();
         }
 
+        public void VisitProcStatementGoto(DMASTProcStatementGoto statementGoto) {
+            _proc.Goto(statementGoto.Label.Identifier);
+        }
+
+        public void VisitProcStatementLabel(DMASTProcStatementLabel statementLabel) {
+            _proc.AddLabel(statementLabel.Name);
```

Comment:
This could end up conflicting with labels generated by other pieces of code. For example, a `label0:` label would conflict with the first label created by `_proc.NewLabelName()`

Reply:
I don't know what exactly I should do about this.

Reply:
You could add something like `"_codelabel"` to the end, like you had before.